sql/src/main/java/io/crate/protocols/postgres/PostgresNetty.java

```diff
@@ -212,18 +210,9 @@ protected void doStop() {
                 serverChannels = null;
             }
         }
-
         if (bootstrap != null) {
-            bootstrap.shutdown();
-            workerExecutor.shutdown();
-            bossExecutor.shutdown();
-
-            try {
-                workerExecutor.awaitTermination(2, TimeUnit.SECONDS);
-                bossExecutor.awaitTermination(2, TimeUnit.SECONDS);
-            } catch (InterruptedException e) {
-                Thread.currentThread().interrupt();
-            }
+            bootstrap.releaseExternalResources();
```

Comment:
I'd prefer calling shutdown() as it is more expressive indicating whats going on despite of that releaseExternalResources() is doing the same as shutdown()